### ISSUE TITLE
Focus can't target if the focusable elements are hidden (display:none)

### ISSUE THREAD

**Issue by sendlo** [makeup-js/makeup-keyboard-trap#6]
Results in a failure when the keyboard focus tries to wrap.

**Comment by ianmcburnie** [comment]
Thanks Mike. Will look into it!

**Comment by ianmcburnie** [comment]
@sendlo I updated makeup-focusables (a dependency) to filter out elements with computed style of display none. Can you try yarn/npm upgrade?

https://github.com/makeup-js/makeup-focusables/commit/5978234c0cfab8c11e8a1312a5c9c867afbd2b33#diff-168726dbe96b3ce427e7fedce31bb0bcR13
